packages/datepickers/src/elements/Datepicker/Datepicker.tsx

```diff
@@ -30,66 +30,41 @@ import { DatepickerContext } from './utils/useDatepickerContext';
 import { StyledMenu, StyledMenuWrapper } from '../../styled';
 
 export interface IDatepickerProps {
-  /**
-   * The selected date to display
-   */
+  /** Sets the selected date */
   value?: Date;
   /**
-   * Returns the parsed date
+   * Handles changes to the selected date
+   *
+   * @param {Date} date The selected date to use
    */
   onChange?: (date: Date) => void;
-  /**
-   * Allows customization of date formatting within input.
-   */
+  /** Formats the date */
   formatDate?: (date: Date) => string;
-  /**
-   * Accepts [all valid Intl locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation)
-   */
+  /** Sets the locale. This accepts [all valid international locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation). */
   locale?: string;
-  /**
-   * Minimum value. Users are still able to manually enter a date below this value.
-   */
+  /** Sets the minimum date value. Note: Users can still manually enter a date below this value. */
   minValue?: Date;
-  /**
-   * Maximum value. Users are still able to manually enter a date above this value.
-   */
+  /** Sets the maximum value. Note: Users can still manually enter a date below this value. */
```

Comment:
@jzempel This leaves me asking what "maximum value" means. Would it be more accurate to say:

`Shows a maximum value, but users can still enter a date above this value`

Also want to double check the use of "below" in the second clause. If there is a set `maxValue`, then the exception would be adding data _above_ the value right?

Reply:
Fixed in latest update

Reply:
@austingreendev please review V's comment here.

Reply:
I'm not sure the updated string is accurate. The Maximum value only affects the data selection UI (the relatively positioned element). Users are able to manually type a previous date which would require separate error handling.

Take a look that at the example shown in https://garden.zendesk.com/components/datepicker#time-window If you type a date earlier than today we show a validation message.

Reply:
Updated in last push.